src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -285,9 +209,6 @@
         .shared-link-field-container {
             flex: 1;
             align-items: center;
-        }
-
-        .text-input-with-copy-button-container {
```

Comment:
this class is base class for the `shared-link-field-container` component. i.e. they represent the same exact component so I combined the blocks